ui/src/App.js

```diff
@@ -68,7 +68,7 @@ class App extends Component {
       // react-select default of "Select..." is shown.
       searchPlaceholderText: "",
       // Whether to show visualization facets or text facets
-      showViz: true,
+      showViz: false,
```

Comment:
Why are we setting this to false? I liked seeing the visualizations by default.

Reply:
Fixed